client/client.go

```diff
@@ -461,17 +464,17 @@ func Configure(logger hclog.Logger, providerConfig interface{}) (schema.ClientMe
 				}
 			})
 		if err != nil {
-			return nil, fmt.Errorf("failed to find disabled regions for account %s. AWS Error: %w", account.AccountName, err)
+			return nil, diags.Add(diag.FromError(fmt.Errorf("failed to find disabled regions for account %s. AWS Error: %w", account.AccountName, err), diag.ACCESS))
```

Comment:
would maybe add a Details to all diag.Access errors to help users on how to fix the issue.

Reply:
Fixed for this one (not sure about the msg), but for `loadOrgAccounts`, since it uses a diverse number of calls, all helper functions inside would need to be refactored to return diags themselves and set the details there.